src/test/java/com/rakovets/course/java/core/practice/lambda_expressions/TestTask01.java

```diff
@@ -0,0 +1,40 @@
+package com.rakovets.course.java.core.practice.lambda_expressions;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.stream.Stream;
+
+public class TestTask01 {
+    Task01 task01 = new Task01();
```

Comment:
@BeforeAll means that the annotated method will be executed before all test methods in the current class. 
But this method has a return type "void".
In case @BeforeAll is used, the created object "task01" is not visible in other methods. 
I don't really understand how to apply @BeforeAll in this case